src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/ConnectionPoolTest/ConnectionPoolTest.cs

```diff
@@ -185,7 +185,11 @@ public static void MaxPoolWaitForConnectionTest(string connectionString)
             ManualResetEventSlim taskAllowedToSpeak = new ManualResetEventSlim(false);
 
             Task waitTask = Task.Factory.StartNew(() => MaxPoolWaitForConnectionTask(newConnectionString, internalConnection, connectionPool, taskAllowedToSpeak));
-            Thread.Sleep(200);
+            int cnt = 5;
```

Comment:
Nit: I'd prefer just naming it `count` or a longer name to make its purpose obvious. There's nothing gained by shortening it.

Reply:
Addressed.